### PR TITLE
Implement react markdown

### PR DESCRIPTION
**Pull request by stefkudla**
Implemented the react-markdown component for parsing and displaying markdown content in the body of articles
- easier to use
- custom component capability
- improved performance for images
- improved security for links

### CODE REVIEW

File: src/pages/works/[slug].jsx [hunk 1]
```diff
@@ -38,7 +37,7 @@ const Post = ({ post, preview }) => {
                 <AlertPreview preview={true} />
               ) : undefined}
               <PostHeader post={post} />
-              <PostBody content={post.content} />
```
Comment (tonyspiro) [inline src/pages/works/[slug].jsx:41]: Same question about the content model in Cosmic for this slug.

**Review by tonyspiro — COMMENTED** [review]
Can you clarify if these changes require any content model updates in Cosmic?

File: src/pages/posts/[slug].jsx [hunk 2]
```diff
@@ -38,7 +37,7 @@ const Post = ({ post, preview }) => {
                 <AlertPreview preview={true} />
               ) : undefined}
               <PostHeader post={post} />
-              <PostBody content={post.content} />
```
[the remark this answers is not in the document]
Comment (stefkudla) [reply src/pages/posts/[slug].jsx:41]: It does not. I didn't have to change anything in Cosmic.

File: src/pages/works/[slug].jsx (hunk 1 above)
Comment (stefkudla) [reply src/pages/works/[slug].jsx:41]: No changes required in Cosmic

**Review by tonyspiro — APPROVED** [review]
LGTM!

**Pull request merged**